pom.xml

```diff
@@ -783,11 +796,11 @@
             <artifactId>graphql-java</artifactId>
             <version>2.2.0</version>
         </dependency>
-        <!-- Provides annotations like XmlElement indicating how classes should be serialized -->
+        <!-- Provides annotations like XmlElement indicating how classes should be serialized  -->
         <dependency>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
-            <version>2.3.1</version>
+            <version>2.3.0</version>
```

Comment:
Is this version decrease intentional?

Reply:
I think this is a merge error - not intentional. @gmellemstrand can you confirm this?

Reply:
This is a merge error. I can fix it.